assets/blocks/lesson-actions/lesson-actions-block/lesson-actions-edit.js

```diff
@@ -68,18 +67,7 @@ const LessonActionsEdit = ( props ) => {
 		: 'not-allowed';
 
 	if ( courseThemeEnabled ) {
-		return (
-			<Notice
-				status="warning"
-				isDismissible={ false }
-				className="wp-block-sensei-lms-lesson-actions__notice"
-			>
-				{ __(
-					'Lesson Actions block is not displayed when Learning Mode is enabled.',
-					'sensei-lms'
-				) }
-			</Notice>
-		);
+		return <></>;
```

Comment:
```suggestion
		return null;
```

Null is the recommended way to prevent a component to be rendered. https://reactjs.org/docs/conditional-rendering.html#preventing-component-from-rendering